.github/workflows/close-pending.yaml

```diff
@@ -0,0 +1,32 @@
+# Licensed to the Apache Software Foundation (ASF) under one or more
+# contributor license agreements.  See the NOTICE file distributed with
+# this work for additional information regarding copyright ownership.
+# The ASF licenses this file to You under the Apache License, Version 2.0
+# (the "License"); you may not use this file except in compliance with
+# the License.  You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+name: close-prs
+
+on:
+  schedule:
+  - cron:  '*/5 * * * *'
```

Comment:
I think it would be better to run this workflow once per day instead of every 5 minutes.  The list of pending PRs to be closed is filtered only by date (eg. 2020-08-03), not time, so most of the runs would be no-ops.

Reply:
Absolutely, I started with one day but changed it to five minutes to show if it works. Changing back.